### ISSUE TITLE
Playoff Question

### ISSUE THREAD

**Issue by HenioH** [aqw/xbmc-gamepass#282]
Hello didn't know where to ask this so asking here. Will the GP addon auto include playoffs? As reg season has ended... I'm assuming since playoffs are determined by reg season and no way to forecast which teams will win.

**Comment by Ghoschdi** [comment]
The GP addon includes whatever is included in your subscription. If you have the season plus package all the games can be watched with the addon.

**Comment by HenioH** [comment]
Yes I have the Season Plus and no wild card or playoffs showing in the addon as of yet. Hoping as they get those schedules out they will end up just showing.

**Comment by kaileu** [comment]
It takes some time till the Section is showing.
We parse the Information from the official app and they didn't add the  
Section yet.

Mit freundlichen Grüßen
Kai Leutner

**Comment by HenioH** [comment]
Thanks for the info Kaileu much appreciate.
